README.md

```diff
@@ -203,14 +200,12 @@ If this option is set to True then meta objects are parsed and searchable up to
 
 *WARNING* If this option is True, your metadata objects MUST have consistent types across all log messages or the metadata object may not be parsed properly.
 
```

Comment:
Use "might" instead o f"may." (The reason is that "may" implies permission and "might"implies chance or possibility.)

Reply:
Done, @tanberry!